packages/react-native-builder-bob/src/index.ts

```diff
@@ -295,9 +295,14 @@ yargs
         },
       };
 
+      if (pkg.codegenConfig && !pkg.codegenConfig.includesGeneratedCode) {
+        // @ts-expect-error The exports is not strictly types therefore it doesn't know about the package.json property
+        exportsField['./package.json'] = './package.json';
```

Comment:
This field should always be present, imo. There are tools, like linters, that reads `package.json` for various reasons.